alacritty/src/logging.rs

```diff
@@ -97,14 +95,14 @@ impl log::Log for Logger {
                         env_var,
                         record.args(),
                     );
-                    let color = match record.level() {
-                        Level::Error => color::RED,
-                        Level::Warn => color::YELLOW,
+                    let message_type = match record.level() {
+                        Level::Error => MessageType::Error,
+                        Level::Warn => MessageType::Warning,
                         _ => unreachable!(),
```

Comment:
You don't have to do it in this PR, but I think we can significantly improve the structure of this method and avoid this `unreachable!` completely.

If we move the logfile writing/stdout/message bar to different methods, we might be able to clean things up a lot easier with match statements and early returns.

I can also send a separate PR to propose some cleanup on this. That might make things a little more clear so the refactoring isn't conflated with the bugfix.

Reply:
yeah, but I'd rather do it separately tbqh.

Reply:
Sounds good, yeah.